datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/group/AddGroupMembersResolver.java

```diff
@@ -98,25 +101,27 @@ private void addUserToGroup(final String userUrnStr, final String groupUrnStr, f
 
   private boolean groupExists(final String groupUrnStr, final QueryContext context) {
     try {
-      final VersionedAspect keyAspect = _entityClient.getAspectOrNull(
-          groupUrnStr,
-          Constants.CORP_GROUP_KEY_ASPECT_NAME,
-          Constants.ASPECT_LATEST_VERSION,
-          context.getAuthentication());
-      return keyAspect != null;
+      Urn groupUrn = Urn.createFromString(groupUrnStr);
+      final EntityResponse entityResponse = _entityClient.batchGetV2(
+          CORP_GROUP_ENTITY_NAME,
+          Collections.singleton(groupUrn),
+          Collections.singleton(CORP_GROUP_KEY_ASPECT_NAME),
+          context.getAuthentication()).get(groupUrn);
+      return entityResponse != null && entityResponse.getAspects().containsKey(CORP_GROUP_KEY_ASPECT_NAME);
     } catch (Exception e) {
       throw new DataHubGraphQLException("Failed to fetch group!", DataHubGraphQLErrorCode.SERVER_ERROR);
     }
   }
 
   private boolean userExists(final String userUrnStr, final QueryContext context) {
     try {
-      final VersionedAspect keyAspect = _entityClient.getAspectOrNull(
-          userUrnStr,
-          Constants.CORP_USER_KEY_ASPECT_NAME,
-          Constants.ASPECT_LATEST_VERSION,
-          context.getAuthentication());
-      return keyAspect != null;
+      Urn userUrn = Urn.createFromString(userUrnStr);
```

Comment:
So I think previously if there was no key aspect, this would getAspectOrNull would return null... But I believe that batchGetV2 artificially injects a key aspect into the result set to maintain our previous behavior. In this particular case it won't be a huge deal but it might be in other cases... Gabe added an "exists" API in GMS that we should migrate this to instead... 

Don't may not need to address in this PR, but definitely want to call it out